manifesto.html

```diff
@@ -27,6 +27,7 @@ <h1 class="cover-heading">Bądźmy życzliwi ❤</h1>
         z drugą stroną upewnić się, że czuje się z tym OK oraz faktycznie spróbować ją
         zrozumieć. Im bardziej delikatny jest temat, tym więcej ostrożności i empatii on
         wymaga. Nie stosuj agresji, w tym biernej i nie odpowiadaj na nią bezpośrednio.
+        Jeśli masz wątpliwości, czy coś jest okej wobec innych, czy też nie – najpierw zapytaj.
```

Comment:
Zdanie z linijek 26-28 również sugeruje zapytanie się społeczności, czy dane zachowanie jest akceptowane.

Reply:
Ale nie wybrzmiewa to aż tak wprost. Nie wiem, nie jestem jakoś mocno przywiązany do tego wniosku, moim zdaniem warto to dodać, ale nic się nie stanie, jak się tego nie zrobi.